landscape/client/broker/tests/test_transport.py

```diff
@@ -196,9 +196,10 @@ def test_ssl_verification_negative(self):
         def got_result(ignored):
             self.assertIs(r.request, None)
             self.assertIs(r.content, None)
+            logfile_value = self.logfile.getvalue()
             self.assertTrue(
-                "server certificate verification failed"
-                in self.logfile.getvalue(),
+                "server certificate verification failed" in logfile_value
+                or "SSL certificate problem" in logfile_value,
```

Comment:
Could we add comments here to help us remember why this test is like this?

Reply:
For sure.